config/cloudinary.yml

```diff
@@ -0,0 +1,19 @@
+---
+development:
+  cloud_name: ddgjr1vra
+  api_key: '449374544161443'
+  api_secret: NGK267mJjV4V1STNxXFGVTdCKUQ
+  enhance_image_tag: true
+  static_file_support: false
+production:
```

Comment:
you should probably use different cloudinary `clouds` for each environment, otherwise, you will end up overwriting some files. But for development and test, you could use the file storage and use cloudinary for production only.

Also, you could try and put the credentials as environment variables for security reasons.